packages/foreman/nodejs-jquery-ujs/nodejs-jquery-ujs.spec

```diff
@@ -1,56 +1,50 @@
 %global npm_name jquery-ujs
+%global enable_tests 1
```

Comment:
Out of curiosity, what does this do? Run tests on build?

Reply:
Yea - enables test to run during build in the `%check` section